torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3127,6 +3165,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_extremal',
                                 active_if=IS_WINDOWS),
                    )),
+    OpInfo('gradient',
+           dtypes=floating_and_complex_types_and(torch.int8, torch.int16,
+                                                 torch.int32, torch.int64,
+                                                 torch.bfloat16, torch.half),
+           supports_out=False,
+           skips=(
+               # following tests give a runtime error with undefined value tensor
+               SkipInfo('TestCommon', 'test_variant_consistency_jit', device_type='cpu',
```

Comment:
Which value is causing the runtime error, and what does the runtime error say?

Reply:
Runtime error is as following : 

RuntimeError: 
Arguments for call are not valid.
The following variants are available:
  
  aten::gradient.scalarint(Tensor self, *, Scalar? spacing=None, int? dim=None, int edge_order=1) -> (Tensor[]):
  Expected a value of type 'Optional[number]' for argument 'spacing' but instead found type 'List[float]'.
  
  aten::gradient.scalararray(Tensor self, *, Scalar spacing, int[] dim, int edge_order=1) -> (Tensor[]):
  Expected a value of type 'number' for argument 'spacing' but instead found type 'List[float]'.
  
  aten::gradient.scalarrayint(Tensor self, *, Scalar[] spacing, int? dim=None, int edge_order=1) -> (Tensor[]):
  Expected a value of type 'List[number]' for argument 'spacing' but instead found type 'List[float]'.
  
  aten::gradient.scalarrayarray(Tensor self, *, Scalar[] spacing, int[] dim, int edge_order=1) -> (Tensor[]):
  Expected a value of type 'List[number]' for argument 'spacing' but instead found type 'List[float]'.
  
  aten::gradient.tensorarrayint(Tensor self, *, Tensor[] spacing, int? dim=None, int edge_order=1) -> (Tensor[]):
  Expected a value of type 'List[Tensor]' for argument 'spacing' but instead found type 'List[float]'.
  
  aten::gradient.tensorarray(Tensor self, *, Tensor[] spacing, int[] dim, int edge_order=1) -> (Tensor[]):
  Expected a value of type 'List[Tensor]' for argument 'spacing' but instead found type 'List[float]'.

The original call is:
  File "<string>", line 3

def the_method(i0):
    return torch.gradient(i0, spacing=[2.0, 2.1])
           ~~~~~~~~~~~~~~ <--- HERE